Triangle: type of Kind is forced to string

If Kind is not string, submitting got this error message:

    # triangle
    ./triangle_test.go:50:5: (*testing.common).Fatalf format %s has arg pair1.kind of wrong type triangle.Kind
    FAIL	triangle [build failed]
    '/usr/local/go/bin/go test --short --json .' returned exit code 2: exit status 2

It seems that recent change of `triangle_test.go` line 50 on https://github.com/exercism/go/pull/2469 causes this issue.

--- a/exercises/practice/triangle/triangle_test.go
+++ b/exercises/practice/triangle/triangle_test.go
@@ -47,7 +47,7 @@ func TestKindsNotEqual(t *testing.T) {
 		for j := i + 1; j < len(kindsAndNames); j++ {
 			pair2 := kindsAndNames[j]
 			if pair1.kind == pair2.kind {
-				t.Fatalf("%s should not be equal to %s", pair1.kind, pair2.name)
+				t.Fatalf("the value of %s should not be equal to the value of %s", pair1.name, pair2.name)
 			}
 		}
 	}

